common/djangoapps/track/views/tests/test_views.py

```diff
@@ -225,28 +222,35 @@ def test_server_track_with_middleware(self):
             views.server_track(request, str(sentinel.event_type), '{}')
 
             expected_event = {
-                'accept_language': '',
-                'referer': '',
-                'username': 'anonymous',
-                'ip': '127.0.0.1',
-                'event_source': 'server',
-                'event_type': str(sentinel.event_type),
-                'event': '{}',
-                'agent': '',
-                'page': None,
-                'time': FROZEN_TIME,
-                'host': 'testserver',
+                'timestamp': FROZEN_TIME,
+                'data': '{"GET": {}, "POST": {}}',
+                'name': self.path_with_course,
                 'context': {
+                    'username': 'anonymous',
                     'user_id': '',
-                    'course_id': u'foo/bar/baz',
```

Comment:
I'm hoping/assuming these tests will fail as written now since the course_id should now be correctly set in the context?

Reply:
Fixed those.